### DISCUSSION TITLE
Troubleshoot Kridosono (Pfsense, Node3, Superserver)

### DISCUSSION THREAD

**Discussion by ardantus** [ardanteam/purchasing#7] [Show and tell]
Agenda1 
Pfsense melakukan penggantian ram 4GB 2pcs, stok gudang ada 4pcs
Untuk meminimalisir saat penggantian membawa type yang beda.

Compute
Menunggu replacement dari ikubaru, sementara barang belum datang menggunakan yang sudah ada, deploy menggunakan compute 234

Node3
Akan dilakukan penggantian untuk skenario pertama, penggantian di site, jika tidak berhasil maka dilakukan penarikan mesin untuk dilakukan troubleshot di uha.

**Comment by nug1e** [comment]
Apakah sudah ada jadwal untuk Agenda 1 ini?

- Maintenance router pfsense
- Deploy compute 234 dan estimasi barang datang dari ikubaru serta proses pengerjaan replacement
- Replace Node3 di site dan plan penarikan Node 3

**Reply by ardantus** [reply]
Jadwalnya dikarenakan ops hardware ada yg cuti kemungkinan baru bisa senin kirim tim ke dacen, agendanya ganti ram pfsense ini 4gb x 2 barang sudah ada, untuk deploy compute 234 itu yg ready saat ini karena compute1 berkurang memorynya sehingga secara compute sudah bisa di gunakan dengan teknis penggunaan di 234 yang 1 nya terakhir karena akan ada replacemenet dari ikubaru.
node3 ganti ram bareng dengan ganti ram pfsense besok senin 17 januari 2022. untuk memory ddrnya sudah ada di gudang.


* update saat ini sudah bisa buat vm di krd, yg running storagenya 124. tapi masalah network pfsense yg sering mati jadi blm bisa di nyatakan running

**Reply by nug1e** [reply]
wokey mas, berarti Agenda 1 ini :

**Senin, 17 Januari 2022**
- [ ] Maintenance router pfsense dengan detail pengerjaan pergantian RAM 2 x 4GB
- [ ] Maintenance Node 3 dengan detail pengerjaan pergantian RAM 

**Persiapan Maintenance ke Dacen, Senin 17 Januari 2022**
- [ ] Koordinasi tim internal, siapa saja yang akan implementasi ke dacen
- [ ] Persiapan berkas dan surat yang perlu disiapkan dan dikoordinasikan dengan pihak internal dan eksternal
- [ ] Persiapan mesin, alat dan perlengkapan yang akan dipergunakan dalam kegiatan maintenance

_note: mgkn perlu didetailkan lagi kebutuhan yang lainnya_

**Estimasi 1 minggu kedepan menunggu update dari inkubaru**
- [ ] Maintenance mesin compute 1 dengan detail pengerjaan replacement server

**Comment by atink1987** [comment]
hari senin untuk PIC nya Atink_Caem dan David, antisipasi kemungkinan terburuk bahwa mesin tidak bisa solved on site, maka node3 akan di bawa ke kntor UHA untuk di lakukan troubleshoot, transport menggunakan stream kantor pukul 10.00 WIB
